script/test_in_openshift/README.md

```diff
@@ -0,0 +1,47 @@
+# Functional tests for Kompose on OpenShift
+
+## Introduction
+
+The functional tests for Kompose on OpenShift cluster leverages  `oc cluster up` to bring a single-cluster OpenShift instance. The test scripts
+are hosted under script/test_in_openshift. The directory structure is as below:
```

Comment:
remove random newline. Or do newline after the `script/test_in_openshift` sentence.